File: src/main/java/org/railwaystations/rsapi/RsapiApplication.java
package org.railwaystations.rsapi;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
import org.springframework.scheduling.annotation.EnableScheduling;
import org.springframework.transaction.annotation.EnableTransactionManagement;

@SpringBootApplication
@ConfigurationPropertiesScan
@EnableScheduling
@EnableTransactionManagement
public class RsapiApplication {

	public static void main(final String[] args) {
		SpringApplication.run(RsapiApplication.class, args);
	}

}

File: service/src/main/kotlin/org/railwaystations/rsapi/RsapiApplication.kt
package org.railwaystations.rsapi

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication
import org.springframework.context.annotation.ComponentScan
import org.springframework.scheduling.annotation.EnableAsync
import org.springframework.scheduling.annotation.EnableScheduling
import org.springframework.transaction.annotation.EnableTransactionManagement

@SpringBootApplication
@ComponentScan("org.railwaystations.rsapi")
@ConfigurationPropertiesScan("org.railwaystations.rsapi")
@EnableScheduling
@EnableAsync
@EnableTransactionManagement
class RsapiApplication

fun main(args: Array<String>) {
    runApplication<RsapiApplication>(*args)
}
